{
  "id": "CVE-2023-36632",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2023-06-25T18:15:09.313",
  "lastModified": "2023-06-29T13:15:09.640",
  "vulnStatus": "Awaiting Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "** DISPUTED ** The legacy email.utils.parseaddr function in Python through 3.11.4 allows attackers to trigger \"RecursionError: maximum recursion depth exceeded while calling a Python object\" via a crafted argument. This argument is plausibly an untrusted value from an application's input data that was supposed to contain a name and an e-mail address. NOTE: email.utils.parseaddr is categorized as a Legacy API in the documentation of the Python email package. Applications should instead use the email.parser.BytesParser or email.parser.Parser class. NOTE: the vendor's perspective is that this is neither a vulnerability nor a bug. The email package is intended to have size limits and to throw an exception when limits are exceeded; they were exceeded by the example demonstration code."
    }
  ],
  "metrics": {},
  "references": [
    {
      "url": "https://docs.python.org/3/library/email.html",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://docs.python.org/3/library/email.utils.html",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://github.com/Daybreak2019/PoC_python3.9_Vul/blob/main/RecursionError-email.utils.parseaddr.py",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://github.com/python/cpython/issues/103800",
      "source": "cve@mitre.org"
    }
  ]
}